shipyard-controller/debug-ui/blocking.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>DebugUI</title>
+    <link rel="icon" type="image/png" href="./favicon.png" />
+
+    <link rel="stylesheet" href="index.css" />
+    <script src="./fetch_blockingsequences.js"></script>
+  </head>
+  <body>
+    <ul class="header">
+      <li class="header">
+        <a href="/">
+          <img src="./favicon.png" alt="" />
+        </a>
+      </li>
+      <li class="header" id="titlecontext"></li>
+    </ul>
+
+    <ul id="blockingSequenceList">
+    </ul>
+
+    <script>
+      const urlParams = new URLSearchParams(window.location.search);
+      const context = urlParams.get("shkeptncontext");
+      const projectname = urlParams.get("projectname");
+      const stage = urlParams.get("stage");
```

Comment:
I would handle the case where one of the query parameters is not set and the value of the `const` is null.
Maybe a null check into `fetchBlockingSequences`